cmds/exp/tsort/tsort.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"io"
+	"strings"
+)
+
+func run(args []string, stdin io.Reader, stdout, stderr io.Writer) error {
+	buf := new(strings.Builder)
+	_, err := io.Copy(buf, stdin)
```

Comment:
I"m a bit concerned about this. You're copying the whole file in? Can't you pull it in a bit at a time?

Reply:
Good catch. I could probably read one line at a time. WDYT?

Reply:
I think that's better. We don't want to give up on systems with "small" -- 64 MiB is small, who knew? -- memory :-)

Reply:
I think this is pretty much done, but feel free to un-resolve this thread and challenge me!